esp-hal/src/soc/esp32s3/gpio.rs

```diff
@@ -70,137 +70,163 @@ pub(crate) fn gpio_intr_enable(int_enable: bool, nmi_enable: bool) -> u8 {
+    SPI3_Q                  = 67,
+    SPI3_D                  = 68,
+    SPI3_HD                 = 69,
+    SPI3_WP                 = 70,
+    SPI3_CS0                = 71,
+    RMT_SIG_0               = 81,
+    RMT_SIG_1               = 82,
+    RMT_SIG_2               = 83,
+    RMT_SIG_3               = 84,
+    I2CEXT0_SCL             = 89,
+    I2CEXT0_SDA             = 90,
+    I2CEXT1_SCL             = 91,
+    I2CEXT1_SDA             = 92,
+    FSPICLK                 = 101,
+    FSPIQ                   = 102,
+    FSPID                   = 103,
+    FSPIHD                  = 104,
+    FSPIWP                  = 105,
+    FSPIIO4                 = 106,
+    FSPIIO5                 = 107,
+    FSPIIO6                 = 108,
+    FSPIIO7                 = 109,
+    FSPICS0                 = 110,
+    TWAI_RX                 = 116,
+    SUBSPIQ                 = 120,
+    SUBSPID                 = 121,
+    SUBSPIHD                = 122,
+    SUBSPIWP                = 123,
+    CAM_DATA_0              = 133,
+    CAM_DATA_1              = 134,
+    CAM_DATA_2              = 135,
+    CAM_DATA_3              = 136,
+    CAM_DATA_4              = 137,
+    CAM_DATA_5              = 138,
+    CAM_DATA_6              = 139,
+    CAM_DATA_7              = 140,
+    CAM_DATA_8              = 141,
+    CAM_DATA_9              = 142,
+    CAM_DATA_10             = 143,
+    CAM_DATA_11             = 144,
+    CAM_DATA_12             = 145,
+    CAM_DATA_13             = 146,
+    CAM_DATA_14             = 147,
+    CAM_DATA_15             = 148,
+    CAM_PCLK                = 149,
+    CAM_H_ENABLE            = 150,
+    CAM_H_SYNC              = 151,
+    CAM_V_SYNC              = 152,
+    SUBSPID4                = 155,
+    SUBSPID5                = 156,
+    SUBSPID6                = 157,
+    SUBSPID7                = 158,
+    SUBSPIDQS               = 159,
+    PWM0_SYNC0              = 160,
+    PWM0_SYNC1              = 161,
+    PWM0_SYNC2              = 162,
+    PWM0_F0                 = 163,
+    PWM0_F1                 = 164,
+    PWM0_F2                 = 165,
+    PWM0_CAP0               = 166,
+    PWM0_CAP1               = 167,
+    PWM0_CAP2               = 168,
+    PWM1_SYNC0              = 169,
+    PWM1_SYNC1              = 170,
+    PWM1_SYNC2              = 171,
+    PWM1_F0                 = 172,
+    PWM1_F1                 = 173,
+    PWM1_F2                 = 174,
+    PWM1_CAP0               = 175,
+    PWM1_CAP1               = 176,
+    PWM1_CAP2               = 177,
+    SDHOST_CCMD_IN_1        = 178,
+    SDHOST_CCMD_IN_2        = 179,
+    SDHOST_CDATA_IN_10      = 180,
+    SDHOST_CDATA_IN_11      = 181,
+    SDHOST_CDATA_IN_12      = 182,
+    SDHOST_CDATA_IN_13      = 183,
+    SDHOST_CDATA_IN_14      = 184,
+    SDHOST_CDATA_IN_15      = 185,
+    SDHOST_CDATA_IN_16      = 186,
+    SDHOST_CDATA_IN_17      = 187,
+    PCMFSYNC                = 188,
+    PCMCLK                  = 189,
```

Comment:
What are these?

Reply:
:shrug: they were there before

Reply:
Ah pf... can you remove those, please, if nothing needs them?

Reply:
Just for completeness they are from ESP-IDF:
```
#define PCMFSYNC_IN_IDX               188
#define PCMCLK_IN_IDX                 189
#define SDHOST_CARD_DETECT_N_1_IDX    194
#define PCMFSYNC_OUT_IDX              194
#define SDHOST_CARD_DETECT_N_2_IDX    195
#define PCMCLK_OUT_IDX                195
```

But only used in Bluedroid for some audio stuff which is only supported by classic bt as it seems - so definitely not needed